README.md

```diff
@@ -0,0 +1,12 @@
+# leaveapplicationsoftware
+Feature
```

Comment:
use second-level header for subheadings.

Reply:
Done